Gui/opensim/view/src/org/opensim/view/actions/EditPreferencesJPanel.java

```diff
@@ -70,8 +70,11 @@
        "AntiAliasingFrames",
        "Marker Display Radius",
        "NonCurrentModelOpacity",
-       "Refresh Rate (ms.)"
-       
+       "Refresh Rate (ms.)",
+       "Muscle Dsiplay Radius",
```

Comment:
Dsiplay -> Display ?

Reply:
@tkuchida I'm retiring the name with typo so users don't see it anymore, the correctly spelled "Muscle Display Radius" will be kept. Thanks for the review :+1:

Reply:
Ok, sounds good. I must be missing something here: GitHub is indicating that you're *adding* "Muscle Dsiplay Radius" which contains the typo.

Reply:
No, you're absolutely right but I'm adding it to the deprecated/hidden list so users who got it by mistake in a previous build don't see it anymore.

Reply:
Ah, ok sorry to hold up the train :steam_locomotive:; I didn't realize there was a list of all the old property names.

Reply:
Unfortunately we need to keep these since registry entries on Windows stay forever! Thanks again for the review 👍